doc/en/user/source/configuration/properties/index.rst

```diff
@@ -203,6 +203,30 @@ GeoServer Property Reference
      - x
      - x
      - x
+   * - GEOSERVER_FORCE_FREEMARKER_ESCAPING
+
+       :doc:`/production/config`, default true
+     - x
+     - x
+     - x
+   * - GEOSERVER_FREEMARKER_ALLOW_LIST
+
+       :doc:`/tutorials/GetFeatureInfo/html`
+     - x
+     - x
+     - x
+   * - GEOSERVER_FREEMARKER_BLOCK_LIST
+
+       :doc:`/tutorials/GetFeatureInfo/html`
+     - x
+     - x
+     - x
+   * - GEOSERVER_FREEMARKER_API_EXPOSED
+
+       :doc:`/tutorials/GetFeatureInfo/html`, default false
```

Comment:
Thanks for updating this tabl.

I agree this is the correct link right now ... but I wonder if there is a more appropriate home for this reference information than a tutorial?